scripts/deployment/goerli-testnets.json

```diff
@@ -1,13 +1,44 @@
 {
   "base_chain": {
     "name": "Goerli",
-    "rpc": "https://goerli.infura.io/v3/9aa3d95b3bc440fa88ea12eaa4456161",
+    "rpc": "",
     "chain_id": 5
   },
   "chains": [
+    {
+      "name": "Goerli",
+      "rpc": "",
+      "chain_id": 5,
+      "finality_period": 1,
+      "transfer_cost": 1000000000000000,
+      "target_weight_ppm": 500000,
+      "l1_messenger": "EthereumL1Messenger",
+      "l2_messenger": [
+        "EthereumL2Messenger",
+        "${resolver}"
+      ],
+      "request_manager_arguments": {
+        "claim_stake": 0.00047,
+        "claim_request_extension": 900,
+        "claim_period": 900,
+        "challenge_period_extension": 900
+      },
+      "fees": {
+        "min_fee_ppm": 300000,
+        "lp_fee_ppm": 0,
+        "protocol_fee_ppm": 0
+      },
+      "tokens": [
+        {
+          "token_address": "mintable_token",
+          "transfer_limit": 10000,
+          "eth_in_token": 1000
+        }
+      ]
+    },
     {
       "name": "Arbitrum Goerli",
-      "rpc": "https://goerli-rollup.arbitrum.io/rpc",
+      "rpc": "",
```

Comment:
Do we really need to remove the rpc urls for the testnets?
All of them are public so i don't see a reason why we would make the job of re-deploying on Goerli harder for us (searching online and finding out which rpc urls we should insert here before deploying).

Reply:
The public RPCs are not stable enough for a deployment always (risk of rate limit). I'd rather add our dev rpcs. 
On the other hand, the change in the deployment infrastructure will change the concept anyway and we have to add an rpc file.